CHANGELOG.rst

```diff
@@ -5,12 +5,31 @@ Here you can see the full list of changes between sqlalchemy-filters
 versions, where semantic versioning is used: *major.minor.patch*.
 
 
+Version 0.9.0
+-------------
+
+Released 2019-03-07
+
+* Add compatibility (no official support) with Python 2.7 (#23 which
+  addresses #18 thanks to @itdependsnetworks)
+* Add support for Python 3.7 (#25)
+* Add support (tests) for PostgreSQL (#28)
+* Add reStructuredText linter (#27)
+* Fix and improve documentation (#21 thanks to @daviskirk, #28)
+* Add ``MANIFEST.in`` file (#27)
+* Tox improvements (#27)
+* Travis CI: use docker, ``xenial`` distribution and Python 3.7 (#28)
+* Improve PyPI classifiers (#25)
```

Comment:
The last four here relate to developer tooling, which probably isn't helpful in the CHANGELOG.

Same thing applies to the reStructuredText linter. It's not a material change that users of the library will want to be aware of.

Reply:
Thanks for the suggestion. I've removed those items from the CHANGELOG.